### ISSUE TITLE
PySide6 exit automatically without showing the window when using FramelessMainWindow

### ISSUE THREAD

**Issue by abcdesteve** [Nuitka/Nuitka#2331]
If you want to post a problem/bug, to help us understand and resolve your issue
please check that you have provided at least the information below, and discard
up to here:

* Nuitka version, full Python version, flavor, OS, etc. as output by *this*
  command.

  > python -m nuitka --version
1.7.4
Commercial: None
Python: 3.10.11 | packaged by Anaconda, Inc. | (main, May 16 2023, 00:55:32) [MSC v.1916 64 bit (AMD64)]
Flavor: Anaconda Python
Executable: D:\miniconda3\envs\default\python.exe
OS: Windows
Arch: x86_64
WindowsRelease: 10
Version C compiler: C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.34.31933\bin\Hostx64\x64\cl.exe (cl 14.3).

* How did you install Nuitka and Python

Python: install from miniconda3
Nuitka: install from pip
virtualenv: true

* The specific PyPI names and versions

  It should be taken from this output if there specific packages involved, e.g.
  `numpy`, you are expected to shorten this to the relevant ones.

  > python -m pip freeze
darkdetect==0.8.0
keyboard==0.13.5
mouse==0.7.1
Nuitka==1.7.4
pyperclip==1.8.2
PySide6==6.5.1.1
PySide6-Addons==6.5.1.1
PySide6-Essentials==6.5.1.1
PySide6-Fluent-Widgets==1.0.1
PySideSix-Frameless-Window==0.1.0
shiboken6==6.5.1.1

* Many times when you get an error from Nuitka, your setup may be special

print("hello world") does work

* Also supply a Short, Self Contained, Correct, Example

```python
import os,sys
from PySide6.QtCore import *
from PySide6.QtGui import *
from PySide6.QtWidgets import *
from qframelesswindow import FramelessMainWindow


class Main(FramelessMainWindow):
    def __init__(self):
        super().__init__()
        self.show()


if __name__ == '__main__':
    app = QApplication(sys.argv)
    win=Main()
    sys.exit(app.exec())
```
qframelesswindow is a lib that provides a fluent design pyside window, you can learn more in https://github.com/zhiyiYo/PyQt-Frameless-Window/tree/PySide6
The program worked if I replace framelessmainwindow with qmainwindow.

* Provide in your issue the Nuitka options used

```
D:\miniconda3\envs\default\python.exe -m nuitka --output-dir="D:\Users\abcdesteve\slsqq\plugins\basic\output" --onefile --windows-disable-console --standalone  --enable-plugin=pyside6          D:\Users\abcdesteve\Desktop\temp.py
Nuitka-Options:INFO: Used command line options: --output-dir=D:\Users\abcdesteve\slsqq\plugins\basic\output --onefile --windows-disable-console --standalone --enable-plugin=pyside6 D:\Users\abcdesteve\Desktop\temp.py
```

* Avoid unnecessary options

  Do not use `--onefile` if the issue also happens with `--standalone`. Minimize
  the use of options as far as you can, please. Do not disable outputs with
  `--quiet` and do not disable warnings.

* Note if this is a regression

  If it used to work in an earlier version of Nuitka, please note what you know
  about that. Since git bisect is a thing for which we do a lot of error to make
  it usable, this will help dramatically to isolate the issue.

**Comment by kayhayen** [comment]
Disabling the console prevents you from seeing error message, don't do that.

My suspect would be that chances are, that qt plugins will have to be added. You can try `--include-qt-plugins=all` for good measure.

**Comment by kayhayen** [comment]
Any feedback on this one, I will have to close it without needed information.

**Comment by kayhayen** [comment]
Closing for lack of feedback.
